components/bl-transitions-component/src/helpers.js

```diff
@@ -25,16 +25,12 @@ export const useTransition = (
       if(readyToStartTransition) {
         setPodElementDimension(podElement['client' + dimensionName]);
         setIsTakenMeasurements(true);
+
+        transitionRef.current.style[dimensionName.toLowerCase()] = '0px';
```

Comment:
`dimensionName.toLowerCase()` looks dirty. Also as `'client' + dimensionName`

Could we get rid of it?